contents/node/nodejs.md

```diff
@@ -31,11 +31,11 @@ Now that we know what Node is, let us look at some benefits it has to offer.
 
 ### Benefit 1: Easy to get Started
 
-To install Node, simply download the installer from the [official Node website](https://nodejs.org/en/download/) based on your OS and run it. This should install both Node and npm. npm is a tool which will help you to search, install and manage node packages. The use of npm will be explained in the next section.
+To install Node, simply download the installer from the [official Node website](https://nodejs.org/en/download/) based on your OS and run it. This should install both Node and npm. Npm is a tool which will help you to search, install and manage node packages. The use of npm will be explained [later](#benefit-5-easy-dependency-management-with-npm).
 
-Here is an example of how to write a working Node server taken from [codeburst](https://codeburst.io/getting-started-with-node-js-a-beginners-guide-b03e25bca71b).
+The following simple example demonstrates how a Node application imports required modules, creates a server to listen to client's request, and then reads the request and returns a response.
```

Comment:
It sounds like this has 3 steps:
1. import stuff
1. create server
1. read request and return response

But step 3 obviously doesn't happen until a request is sent. Maybe try to rephrase?
```suggestion
The following simple example demonstrates how a Node application imports required modules, creates a server to listen for clients' requests, and then reads the request and returns a response.
```